install_config/persistent_storage/persistent_storage_gce.adoc

```diff
@@ -142,3 +142,35 @@ formatted with the given file system.
 
 This allows using unformatted GCE volumes as persistent volumes, because
 {product-title} formats them before the first use.
+
```

Comment:
add this ID before the heading:
[[gce-persistent-disk-multi-zone-configuration]]